drivers/isl29125/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = isl29125
```

Comment:
Just for the protocol: looking at different sensor-driver makefiles I see that this point seems to be unclear. I'll make a note in my todo list.